index.html

```diff
@@ -0,0 +1,202 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script
+      src="https://kit.fontawesome.com/cb6c5078b1.js"
+      crossorigin="anonymous"
+    ></script>
+    <link
+href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="styles.css" />
+    <title>The Burning Circle Airshow 2021</title>
+  </head>
+  <body>
+    <div class="top-section">
+      <div class="bar-container">
+        <ul class="bars">
+          <li>
+            <i class="fab fa-facebook-f"></i>
+          </li>
+          <li>
+            <i class="fab fa-twitter"></i>
+          </li>
+          <li>
+            <h4>English</h4>
+          </li>
+          <li>
+            <h4>My Page</h4>
+          </li>
+          <li>
+            <h4>Logout</h4>
+          </li>
+        </ul>
+      </div>
+    </div>
+    <header>
+      <div class="navbar">
+          <div class="logo-DT">
+            <a href="index.html">BCFA</a>
+            <div class="menu-bar nav" id="menu">
+              <i class="fas fa-bars fa-3x"></i>
```

Comment:
* I suggest you make the menu bar more visible this would enable users to see the menu bar easily Hint 💭: you can make the background lighter so the menu bar would be more visible 
![image](https://user-images.githubusercontent.com/52497246/133898259-e4295106-4dfb-43ed-a36e-2886e7ee0afe.png)

Reply:
done